boring-sys/build.rs

```diff
@@ -205,9 +205,12 @@ fn main() {
         cfg.build_target("bssl").build().display().to_string()
     });
 
-    let build_path = get_boringssl_platform_output_path();
-    let build_dir = format!("{}/build/{}", bssl_dir, build_path);
-    println!("cargo:rustc-link-search=native={}", build_dir);
+    println!("cargo:rerun-if-env-changed=BORING_BSSL_LIB_PATH");
+    let lib_path = std::env::var("BORING_BSSL_LIB_PATH").unwrap_or_else(|_| {
```

Comment:
Why this change? What are you using it for?

Reply:
In my build environment, the I must set the include path and library path to different values, or example:
```
$ env | grep BORING                                  
BORING_BSSL_INCLUDE_PATH=/opt/boringssl-fips/include
BORING_BSSL_LIB_PATH=/opt/boringssl-fips
```
Only then will it successfully build and link with my FIPS build of BoringSSL:
```
$ cargo run --features fips-3678 --example fips_enabled
   Compiling boring-sys v1.1.1 (/home/braden/rust/boring/boring-sys)
   Compiling boring v1.1.6 (/home/braden/rust/boring/boring)
    Finished dev [unoptimized + debuginfo] target(s) in 18.00s
     Running `/home/braden/rust/boring/target/debug/examples/fips_enabled`
boring::fips::enabled(): true
```
Open to any suggestions to simply things or make it work better in other use cases, though.